runelite-client/src/main/java/net/runelite/client/plugins/gabplugs/goldrush/GabulhasGoldRushConfig.java

```diff
@@ -0,0 +1,47 @@
+package net.runelite.client.plugins.gabplugs.goldrush;
```

Comment:
Please put all your files under plugins/Microbot

This is easier to manage all the microbot specific plugins